popupModal.js

```diff
@@ -0,0 +1,128 @@
+    year: '2018',
+    company: 'Lead Developer',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Mulit-Post Stories',
+    imageLink: 'assets/snapshoot-portfolio3.png',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    role: 'Full Stack Dev',
+    technologies: ['html', 'css', 'javascript'],
+    year: '2015',
+    company: 'Facebook',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Tonic',
+    imageLink: 'assets/snapshoot-portfolio4.png',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    role: 'Back End Dev',
+    technologies: ['html', 'css', 'javascript', 'Ruby'],
+    year: '2015',
+    company: 'Canopy',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+];
+
+(() => {
+  projectSecD.innerHTML = '';
+  for (let i = 0; i < projectObject.length; i += 1) {
+    let technologies = '';
+    for (let j = 0; j < projectObject[i].technologies.length; j += 1) {
+      technologies = `${technologies}<li class="project_tag">${projectObject[i].technologies[j]}</li>`;
+    }
+    projectSecD.innerHTML = `${projectSecD.innerHTML}
+    <li class="project">
+    <img src="${projectObject[i].imageLink}" alt="Project Image 1" />
+    <div class="project-des">
+      <h2 class="project-title">${projectObject[i].name}</h2>
+      <div class="projectInformation">
+        <p class="projectCanopy">${projectObject[i].company}</p>
+        <ul class="projectJob">
+          <li class="jobInformation">${projectObject[i].role}</li>
+          <li class="jobInformation">${projectObject[i].year}</li>
+        </ul>
+      </div>
+      <p class="project_full_desc">
+       ${projectObject[i].description}
+      </p>
+      <div class="project_tags_container">
+        <ul class="tags_flex">
+         ${technologies}
+        </ul>
+      </div>
+      <button type="button" class="view_project_btn proj_${i + 1}">
+        See Project
+      </button>
+    </div>
+    </li>`;
+  }
+  for (let i = 0; i < projectObject.length; i += 1) {
+    document.querySelector(`.proj_${i + 1}`).addEventListener('click', () => {
+      titleDis.innerHTML = projectObject[i].name;
+      cardCon.innerHTML = projectObject[i].company;
+      cardRole.innerHTML = projectObject[i].role;
+      cardYear.innerHTML = projectObject[i].year;
+      innerDis.innerHTML = projectObject[i].description;
+      projectImage.src = projectObject[i].imageLink;
+      buttonViewSource.action = projectObject[i].sourceLink;
+      buttonLiveSource.action = projectObject[i].demoLink;
```

Comment:
- [ ] [OPTIONAL] Great job making changes requested by the previous reviewer 💯, but I think it could be better if you fix the "See source" button to stick as much as possible with the original design. 🥇

![image](https://user-images.githubusercontent.com/91301423/202859957-b14c7a1e-a194-4710-924e-971328fcb682.png)